### ISSUE TITLE
Stylelint and other dependencies seems to be requiring an update

### ISSUE THREAD

**Issue by evilaliv3** [wikimedia/grunt-stylelint#118]
Hello!

I just noticed this while updating the dependencies on @globaLeaks

This is the current errors that are reported by npm when trying to install grunt-eslint:
```bash
npm info using npm@9.5.0
npm info using node@v19.7.0
npm http fetch GET 200 https://registry.npmjs.org/stylelint 1800ms (cache revalidated)
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: grunt-stylelint@0.18.0
npm ERR! Found: stylelint@15.2.0
npm ERR! node_modules/stylelint
npm ERR!   dev stylelint@"15.2.0" from the root project
npm ERR!   peer stylelint@"^15.0.0" from stylelint-config-recommended@10.0.1
npm ERR!   node_modules/stylelint-config-recommended
npm ERR!     stylelint-config-recommended@"^10.0.1" from stylelint-config-standard@30.0.1
npm ERR!     node_modules/stylelint-config-standard
npm ERR!       dev stylelint-config-standard@"30.0.1" from the root project
npm ERR!   1 more (stylelint-config-standard)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer stylelint@"14.x" from grunt-stylelint@0.18.0
npm ERR! node_modules/grunt-stylelint
npm ERR!   dev grunt-stylelint@"0.18.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: stylelint@14.16.1
npm ERR! node_modules/stylelint
npm ERR!   peer stylelint@"14.x" from grunt-stylelint@0.18.0
npm ERR!   node_modules/grunt-stylelint
npm ERR!     dev grunt-stylelint@"0.18.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /home/evilaliv3/.npm/_logs/2023-03-16T18_28_36_875Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/evilaliv3/.npm/_logs/2023-03-16T18_28_36_875Z-debug-0.log
```

I already retested at least the update to stylelint 15.2.0 that seems to work like a charm!

Thank you if you could look into this.

**Comment by HolgerJeromin** [comment]
We as users can use an current stylelint with an [override](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides) in your package.json:

```json
{
    "devDependencies": {
        "grunt-stylelint": "0.18.0",
        "stylelint": "15.9.0",
        "stylelint-config-standard": "33.0.0"
    },
    "overrides": {
        "grunt-stylelint": {
            "stylelint": "14.x || 15.x"
        }
    }
}

**Comment by jdforrester** [comment]
Yeah, stylelint 15's dropping of node 12 and node 14 support held us back, but I think this is safe to do now.

**Comment by jdforrester** [comment]
I've published version 0.19.0 with this change in.
